### PR TITLE
Add Composer support

### PR DESCRIPTION
**Pull request by mundschenk-at**
Make a Composer package out of Gamajo-Dashboard-Glancer for ease of use in new plugins.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gamajo/dashboard-glancer",
+  "description": "A class for your WordPress plugin, to make adding custom post type counts to the At a Glance dashboard widget considerably easier.",
+  "keywords": ["wordpress", "templates"],
+  "homepage": "http://github.com/GaryJones/Gamajo-Dashboard-Glancer",
```
Comment (GaryJones) [inline composer.json]: `https` please.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "license": "GPL-2.0-or-later",
+  "authors": [
+    {
+      "name": "Gary Jones",
+      "homepage": "http://gamajo.com",
```
Comment (GaryJones) [inline composer.json]: `https` please.

**Review by GaryJones — CHANGES_REQUESTED** [review]
Couple of minor tweaks, then it would be good to go. Thanks!

**Comment by mundschenk-at** [comment]
Yes, of course this should be `https`. I'll fix the protocol tonight. (On a sidenote, you should probably check `gamajo/template-loader` for this as well, as that was my template for the `composer.json`.)

**Comment by GaryJones** [comment]
> On a sidenote, you should probably check gamajo/template-loader for this as well, as that was my template for the composer.json

https://github.com/GaryJones/Gamajo-Template-Loader/commit/6129b825a81e9a55fa9ccdb72eea307d544f864b, thanks!

**Comment by mundschenk-at** [comment]
Done :)

**Review by GaryJones — APPROVED** [review]

**Comment by GaryJones** [comment]
Thank you :-)

**Comment by mundschenk-at** [comment]
@GaryJones Could you please also add the package on `packagist.org`? Then I can delete the custom repository from my project's `composer.json`.

**Comment by GaryJones** [comment]
@mundschenk-at I've moved this repo from `GaryJones` account on GitHub to `Gamajo`, and renamed it to match the Composer package name.

I've then registered it as https://packagist.org/packages/gamajo/dashboard-glancer.

When I get chance, I may update the file-level DocBlocks, but have no plans for changing anything code-wise.

**Comment by mundschenk-at** [comment]
Thanks!

**Pull request merged**